lib/routes/hypergryph/arknights/arktca.ts

```diff
@@ -0,0 +1,122 @@
+import { Route } from '@/types';
+import got from '@/utils/got';
+import { load } from 'cheerio';
+import { parseDate } from '@/utils/parse-date';
+import cache from '@/utils/cache';
+
+const rssDescription = `期刊《回归线》 | 泰拉创作者联合会`;
+const Url = 'aneot.arktca.com';
+const Author = `Bendancom`;
+
+export const route: Route = {
+    path: '/arknights/arktca',
+    categories: ['game'],
+    example: '/hypergryph/arknights/arktca',
+    parameters: {},
+    features: {
+        requireConfig: false,
+        requirePuppeteer: false,
+        antiCrawler: false,
+        supportBT: false,
+        supportPodcast: false,
+        supportScihub: false,
+    },
+    name: '期刊',
+    url: String(Url),
+    maintainers: [String(Author)],
+    radar: [
+        {
+            source: [String(Url)],
+        },
+    ],
+    description: String(rssDescription),
```

Comment:
https://github.com/DIYgod/RSSHub/blob/adf39eee2211eef816d41e2b9418d10b8a518bd6/lib/routes/hypergryph/arknights/arktca.ts#L8
`Url` is already a `string`. Casting a string with `String()` is unnecessary.